_posts/2020-04-16-remote-junior-developer-tips.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: 'Tips From One Remote Junior Developer to Another.'
+date: 2020-04-19 10:30:00
+categories: ["junior", "remote"]
```

Comment:
@FionaDL Let's use `learning` instead since we already have a few blogposts in that category

```suggestion
categories: ["learning", "remote"]
```